torchci/lib/bot/mergeBot.ts

```diff
@@ -136,7 +141,7 @@ function mergeBot(app: Probot): void {
       case "merge":
         return await handleMerge(args.force, args.green, args.all_green);
       case "rebase":
-        return await handleRebase(args.stable);
+        return await handleRebase(args.stable, args.branch);
```

Comment:
nit: functions shouldn't have redundant arguments; both these args control the branch selection. We should just have one argument (`branch`) and pass it different values depending on what the user wanted.